meson.build

```diff
@@ -888,7 +888,7 @@ conf.set('FOREIGN_UID_BASE', foreign_uid_base)
 nobody_user = get_option('nobody-user')
 nobody_group = get_option('nobody-group')
 
-if not meson.is_cross_build()
+if not meson.is_cross_build() and not get_option('native-tools-only')
```

Comment:
Hmm, all checks below are optional. E.g. if `getent passwd` fails, we skip the check for name of the nobody user. Also, the checks below just warn about spurious config. You can ignore them.

Could you elaborate more ? What do you want to suppress??

Reply:
In order for `getent` to fail it should be present. That's not the case for the default OE build environment. Perhaps checking for its existence first would do the trick?

Reply:
yeah that sounds fine, use `find_program` or so

Reply:
Ah, I see. Yeah. please check the existence with `find_program()` before calling `getent` and friends.